### DISCUSSION TITLE
Issues records are not available in the original repository, could they be exported/imported somehow?

### DISCUSSION THREAD

**Discussion by mik1234mc** [brianguenter/OpticSim.jl#14] [General]
Dear all,

I have a question if there is a plan to export the Issues from original repository area (https://github.com/microsoft/OpticSim.jl) to this repository and import them into this project area. As the old repository is read-only, the Issues are not displayed to me. I don't know if you have an access to that repo anymore, but there were multiple valuable discussions which would be nice to keep for reference or further development.

BR,
Michael

**Comment by brianguenter** [comment]
@mik1234mc I don't have access to that repo anymore. I know people still at Microsoft who do - it's possible they can grab the issues discussions. But I think this is a long shot, unfortunately.
